pythonforandroid/build.py

```diff
@@ -374,7 +365,14 @@ def prepare_build_environment(self,
         if not self.ccache:
             info('ccache is missing, the build will not be optimized in the '
                  'future.')
-        self.cython = get_cython_path()
+        try:
+            subprocess.check_output([
+                "python3", "-m", "cython", "--help",
+            ])
+        except subprocess.CalledProcessError:
+            warning('Cython for python3 missing. If you are building for '
+                    ' a python 3 target (which is the default)'
+                    ' then THINGS WILL BREAK.')
```

Comment:
If things would definitely break. Then let's not be gentle and let's fail fast with `BuildInterruptingException`, what do you guys think?

Reply:
Only if it's a python 3 target. We still support python 2, and at this stage the recipes aren't known :woman_shrugging: so as the text says, it would break but only for a python 3 build. I did some poking around the code and I found no way to tell this in this function

Reply:
Oh Python2, right... Thanks I almost forgot about this folk :joy: